File: app/src/main/java/eu/toma/dev/playground/warcraft/model/item/ItemSpell.java
package eu.toma.dev.playground.warcraft.model.item;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ItemSpell {

    @SerializedName("spellId")
    @Expose
    public Integer spellId;
    @SerializedName("spell")
    @Expose
    public Spell spell;
    @SerializedName("nCharges")
    @Expose
    public Integer nCharges;
    @SerializedName("consumable")
    @Expose
    public Boolean consumable;
    @SerializedName("categoryId")
    @Expose
    public Integer categoryId;
    @SerializedName("trigger")
    @Expose
    public String trigger;

}

File: app/src/main/kotlin/warcraft/model/item/ItemSpell.kt
package warcraft.model.item

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ItemSpell {

    @SerializedName("spellId")
    @Expose
    var spellId: Int? = null
    @SerializedName("spell")
    @Expose
    var spell: Spell
    @SerializedName("nCharges")
    @Expose
    var nCharges: Int? = null
    @SerializedName("consumable")
    @Expose
    var consumable: Boolean? = null
    @SerializedName("categoryId")
    @Expose
    var categoryId: Int? = null
    @SerializedName("trigger")
    @Expose
    var trigger: String

}